components/camera/videosource/static.go

```diff
@@ -73,6 +73,16 @@ type fileSourceConfig struct {
 	PointCloud           string                             `json:"pointcloud_file_path,omitempty"`
 }
 
+// Validate ensures all parts of the config are valid.
+func (c fileSourceConfig) Validate(path string) ([]string, error) {
+	if c.CameraParameters.Width < 0 || c.CameraParameters.Height < 0 {
+		return nil, fmt.Errorf(" webcam width %v, and height %v must be non-zero and positive",
```

Comment:
CameraParameters are from the intrinsics right? Should we mention intrinsics explicitly in the error message?

Also should we change `webcam` -> `image_file cam` or something

Reply:
done, I'm struggling with the phrasing, which I copied from aligns. Do we like a specific phrasing more out of the possible error messages here?

Reply:
likely running make lint-go with my machine's go triggered the addition.